lib/board.rb

```diff
@@ -23,9 +23,18 @@ def valid_coordinate?(coord)
   end
 
   def valid_placement?(ship, coord)
-     coord.length == ship.length
-  end
+    valid = false 
+    #Rationale: If none of the horizontal || vertical rules apply, then not valid
+
+    #rules to place horizontally: letter same, sequintial nums == ship length
+    valid = true if coord.all? { |item| item[0] == coord.first[0] } &&
```

Comment:
I wasn't sure what to call an individual coordinate 😅 so I said item. I am open to suggestions!!!